hollow-jar/full/src/main/java/org/wildfly/swarm/ts/hollow/jar/full/EjbPassivationServlet.java

```diff
@@ -0,0 +1,37 @@
+package org.wildfly.swarm.ts.hollow.jar.full;
+
+import javax.inject.Inject;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.wildfly.swarm.ts.hollow.jar.full.ejb.passivation.EjbPassivationBean;
+
+import java.io.IOException;
+
+@WebServlet("/ejbpassivation")
+public class EjbPassivationServlet extends HttpServlet {
+
+    @Inject
+    private ProcessResult processResult;
```

Comment:
We don't know the order of the tests here, so don't know what can be here at the beginning and if we don't affect other tests that are following. Should we clear the `ProcessResult` at the beginning and at the end of this test?

Reply:
Sure, I was gonna do it but didn't in the end. I think it will make everything more robust.

Reply:
@Ladicek pushed that change you suggested.

Reply:
I did just for the beginning on each test. Maybe that's enough?

Reply:
We probably do it at the beginning of each test? If not, we should, could you please verify?